src/Tizen.Network.Stc/Tizen.Network.Stc/StcInfo.cs

```diff
@@ -0,0 +1,281 @@
+            if (ret != (int)StcError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to get Interface type from info, Error - " + (StcError)ret);
+            }
+            if (ifaceType == NativeNetworkInterface.Unknown)
+            {
+                throw new InvalidOperationException("Interface Type is Unknown.");
+            }
+            return (NetworkInterface)ifaceType;
+        }
+
+        /// <summary>
+        /// A property to get incoming counter from statistics information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>incoming counter.</value>
+        public long IncomingCounter
+        {
+            get
+            {
+                long incoming;
+                long outgoing;
+                int ret = Interop.Stc.Info.GetCounter(_infoHandle, out incoming, out outgoing);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get incoming counter from info, Error - " + (StcError)ret);
+                }
+                return incoming;
+            }
+        }
+
+        /// <summary>
+        /// A property to get outgoing counter from statistics information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>outgoing counter.</value>
+        public long OutgoingCounter
+        {
+            get
+            {
+                long incoming;
+                long outgoing;
+                int ret = Interop.Stc.Info.GetCounter(_infoHandle, out incoming, out outgoing);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get outgoing counter from info, Error - " + (StcError)ret);
+                }
+                return outgoing;
+            }
+        }
+
+        /// <summary>
+        /// A property to get the roaming type from statistics information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>IsRoaming.</value>
+        public bool IsRoaming
+        {
+            get
+            {
+                RoamingType roaming;
+                int ret = Interop.Stc.Info.GetRoaming(_infoHandle, out roaming);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get Roaming type from info, Error - " + (StcError)ret);
+                }
+
+                return roaming == RoamingType.Enabled;
+            }
+        }
+
+        /// <summary>
+        /// To get the network protocol type from statistics information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <returns>NetworkProtocol type.</returns>
+        /// <exception cref="InvalidOperationException">Thrown when method fails due to an invalid operation/value.</exception>
+        public NetworkProtocol GetProtocol()
```

Comment:
Is it possible scenario that throw exception?
Could you explain which case could be occurred error?

Acutally, I want to change it to getter only property without excpetion

Reply:
As suggested in the discussion: Exception throw is removed, Unknown is passed to app as 'Others' and made it a Property.